examples/pod_disruption_scenarios.yaml

```diff
@@ -0,0 +1,469 @@
+tests:
+  - name: krkn-hub-*
+    index: krkn-telemetry*
+    benchmarkIndex: krkn-metrics*
+    version: cluster_version
+    uuid_field: run_uuid
+    metadata:
+      cluster_version: "{{ version }}"
+      scenarios.scenario_type: pod_disruption_scenarios
+      cloud_infrastructure: AWS  # Options: AWS, IBMCloud, GCP, Azure, VSphere
+      cloud_type: self-managed   # Options: self-managed, rosa
+      total_node_count: 9
+      node_summary_infos.instance_type: "*xlarge*"  # Matches: m6a.xlarge (masters), r5.xlarge (workers)
+      network_plugins: OVNKubernetes
+      scenarios.scenario: "*pod_scenario.yaml"
```

Comment:
since we have the scenario_type already, think this variable is over kill

Reply:
Yes, true, I added it, as we need it for the hog scenarios, as all for all hog scenarios, the scenarios.scenario_type is hog_scenarios, hence we need to use scenarios.scenario. We can definitely remove them from the other files